src/app/modules/core/core.component.html

```diff
@@ -0,0 +1,47 @@
+<!-- ********** start ********* -->
```

Comment:
sir its side nav component. all our app component are child of this component. in this component we have side menu link like Dashboard, inbox, user activity , setting , logout